css/style.css

```diff
@@ -0,0 +1,287 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  min-width: 1170px;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 1030px;
+}
+
+/*Первый блок*/
+
+.heading {
+  min-height: 96px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.heading__nav-wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.heading__logo {
+  margin-right: 45px;
+}
+
+.navigation__link {
+  text-decoration: none;
+  font-family: Gotham Pro Bold;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  color: #000000;
+  margin-right: 35px;
+}
+
+.search-form__input {
+    height: 20px;
+    width: 185px;
+    border-bottom: 1px solid #000000;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)